doc/releases/release-notes-3.3.rst

```diff
@@ -1031,6 +1031,8 @@ Networking
 * Sockets:
 
   * Fixed a possible memory leak on failed TLS socket creation.
+  * Socketpair is now marked as supported.
```

Comment:
I think it's a bit too late for such changes for 3.3 release.

Reply:
Yeah, let's wait until after 3.3, just because there is a bit of a cleanup PR in the pipes.

Reply:
@cfriedt It seems that the cleanup PR is now merged, but the experimental tag was not removed. Is it ok for you to remove it here?

Reply:
I think we should get https://github.com/zephyrproject-rtos/zephyr/issues/54996 fixed first.